packages/odyssey-react-mui/src/MenuButton.tsx

```diff
@@ -110,13 +148,25 @@ const MenuButton = ({
 
   const isOpen = Boolean(anchorEl);
 
-  const closeMenu = useCallback<MenuContextType["closeMenu"]>(() => {
-    setAnchorEl(null);
-  }, []);
+  const closeMenu = useCallback(
+    (event?, reason?) => {
+      if (preventCloseOnChildClick && reason !== "backdropClick") {
+        return;
+      }
```

Comment:
Why is this an optional thing? What's the reasoning behind sometimes closing or not closing the menu based on if a child was clicked? Is this so you can open sub-menus?